arcade/__init__.py

```diff
@@ -140,6 +140,7 @@ def configure_logging(level: int = None):
 from .geometry_generic import rotate_point
 from .geometry_generic import clamp
 from .geometry_generic import get_angle_degrees
+from .geometry_generic import get_angle_radians
```

Comment:
It should also be  in `__all__` for

Reply:
There's a comment above `__all__` that says it's generated?

Reply:
Remove that comment. It's no longer generated because it has too manu issues,

Reply:
Done and pushed.